Documentation/source/Getting started.md

```diff
@@ -45,3 +46,115 @@ let documentService: AccountingDocumentService = giniPayApiLib.documentService()
 ```
 
 You are all set 🚀! You can start using the Gini API through the `documentService`.
+
+### Upload a document
+
+As the key aspect of the Gini API is to provide information extraction for analyzing documents, the API is mainly built around the concept of documents. A document can be any written representation of information such as invoices, reminders, contracts and so on.
+
+The Gini Pay API lib supports creating documents from images, PDFs or UTF-8 encoded text. Images are usually a picture of a paper document which was taken with the device’s camera.
+
+The following example shows how to create a new document from a byte array containing a JPEG image.
+```swift
+documentService.createDocument(fileName: "myFirstDocument.jpg",
+                            docType: docType,
+                            type: .partial(document.data),
+                            metadata: metadata) { result in
+    switch result {
+    case let .success(createdDocument):
+        print("📄 Created document with id: \(createdDocument.id) " +
+            "for vision document \(document.id)")
+        completion(.success(createdDocument))
+    case let .failure(error):
+        print("❌ Document creation failed: \(error)")
+        completion(.failure(error))
+    }
+}
+```
+
+Each page of a document needs to uploaded as a partial document. In addition documents consisting of one page also should be uploaded as a partial document.
+
+> ⚠️  **Warning**
+> - If you are using the Gini Accounting API, then partial documents are not supported. Use the `documentService.createDocument()` methods instead.
```

Comment:
This is not needed anymore. The accounting api supports partial documents now.

Reply:
Great, I've copied it from Android documentation, I think it is still there.